ironic/resource_ironic_node_v1.go

```diff
@@ -145,6 +148,25 @@ func resourceNodeV1() *schema.Resource {
 				// This did not change if the current provision state matches the target
 				DiffSuppressFunc: targetStateMatchesReality,
 			},
+			"power_state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"target_power_state": {
+				Type:     schema.TypeString,
+				Optional: true,
+
+				// If power_state is same as target_power_state, we have no changes to apply
```

Comment:
This is only triggered when target_power_state is changed, right?

Reply:
Yup